rootex/core/renderer/shaders/god_rays_pixel_shader.hlsl

```diff
@@ -0,0 +1,63 @@
+#define THRESHOLD 0.95f
+
+cbuffer cbGodRays : register(b0)
+{
+	float3 sunScreenSpacePos;
+	int numSamples;
+	float density;
+	float weight;
+	float decay;
+	float exposure;
+	float2 screenDims;
```

Comment:
I don't think this is specific to god rays, and probably some other CB already sends this data